docusaurus.config.js

```diff
@@ -135,12 +134,20 @@ const config = {
         style: 'dark',
         links: [
           {
-            title: 'Docs',
+            title: 'Our Initiatives',
             items: [
               {
                 label: 'Tutorial Series',
                 to: '/docs/tutorials',
               },
+              {
+                label: 'Nexus Labs',
+                to: '/docs/nexus-labs',
```

Comment:
Suggest renaming `docs/` folder to `our-initiatives` , and changing the corresponding paths in the docasaurus config file so the url matches the section name

Reply:
Good idea! Renamed in content files and rerouted! 🎉